.goreleaser-nightly.yml

```diff
@@ -6,18 +6,31 @@ builds:
       - darwin
     goarch:
       - amd64
-
+      - arm64
+      - arm
+    goarm:
+      - 6
+      - 7
+    ignore:
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: arm
+        goarm: 6
+      - goos: darwin
+        goarch: arm
+        goarm: 7
     main: ./cmd/influx/
     flags:
-      - -tags={{if and (eq .Os "linux") (eq .Arch "amd64")}}osusergo,netgo,static_build{{end}}
+      - -tags={{if eq .Os "linux"}}osusergo,netgo,static_build{{end}}
```

Comment:
Before merge: I think we might need to set `noasm` in the tags here for non-amd64 `GOARCH`, if this bypasses the `Makefile`